### PR TITLE
Latest developments

### PR DESCRIPTION
**Pull request by alexarchambault**
Should make the kernel work in nteract in particular.

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
 addSbtPlugin("org.xerial.sbt" % "sbt-pack" % "0.8.0")
-addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-M15")
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-RC1")
```
Comment (rgbkrk) [inline project/plugins.sbt:3]: Sweet to see you've got a release candidate for coursier!!!

**Comment by rgbkrk** [comment]
What was the issue it was running into?

**Comment by alexarchambault** [comment]
You mean for nteract? https://github.com/alexarchambault/jupyter-kernel/pull/14/commits/6635826313a4a737820e37ea5225102c2f7139d6 fixed it for me. The UI of nteract was just saying "Starting" in its status - and refusing to run any command because of it apparently.

File: project/plugins.sbt (hunk 1 above)
Comment (alexarchambault) [reply project/plugins.sbt:3]: And hopefully 1.0 final soon :-)

**Pull request merged**